torchvision/models/detection/faster_rcnn.py

```diff
@@ -142,8 +142,10 @@ class FasterRCNN(GeneralizedRCNN):
 
     def __init__(self, backbone, num_classes=None,
                  # transform parameters
+                 transform=None,
                  min_size=800, max_size=1333,
-                 image_mean=None, image_std=None,
+                 image_mean=[0.485, 0.456, 0.406],
+                 image_std=[0.229, 0.224, 0.225],
```

Comment:
Can you revert those changes? It's in general preferable not to pass mutable objects as arguments to functions.